templates/model.txt

```diff
@@ -12,7 +12,7 @@ module.exports = mongoose.model('<%= table %>', schema, '<%= table %>');
 %>module.exports = (sequelize, DataTypes) => {
   const { Sequelize } = sequelize;
   const Model = sequelize.define('<%= table %>', {<% _.each(fields, (field) => { %>
-    '<%= field.name %>': {
+    <%= field.cameCaseName %>: {
```

Comment:
I am pretty sure, this change breaks the behaviour for schema without underscores but having field names like this `recordID`:
<img width="402" alt="Screenshot 2019-08-06 at 17 24 40" src="https://user-images.githubusercontent.com/664261/62553390-898b5000-b86f-11e9-8cec-e8773a1c4720.png">